applications/luci-app-throughput-measurements/root/usr/libexec/rpcd/luci.tputmeas

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env lua
+
+local ubus = require "ubus"
+local uci = require "luci.model.uci"
+local nixio = require "nixio"
+local fs = require "nixio.fs"
+local json = require "luci.jsonc"
+local sys = require "luci.sys"
+local posix = require "posix"
+
+
+local methods = {
+	iperf3Start = {
+		args = { mode = "mode", server_address = "server_address", interface = "interface",
+				interface_address = "interface_address", port = "port", duration = "duration",
+				bytes = "bytes", udp = "udp",  bitrate = "bitrate", reverse = "reverse",
+				connections = "connections", interval = "interval" },
+		call = function(args)
+			local iperf3_cmd = "iperf3 --forceflush "
+
+			if fs.stat("/var/run/iperf3.pid") then
+				return { result = "error: Iperf3 is running" }
```

Comment:
Instead of replaying `{ "result": "error: ..." }` I suggest to reply with `{ "error": "..." }`